src/Language/PureScript/Externs.hs

```diff
@@ -73,7 +73,7 @@ data ExternsFixity = ExternsFixity
   -- | The operator symbol
   , efOperator :: String
   -- | The value the operator is an alias for
-  , efAlias :: Maybe Ident
+  , efAlias :: Maybe (Maybe ModuleName, Ident)
```

Comment:
I'm not really sure why I had to do this. I got an error from the `deriveJSON` macro when trying to use `efAlias :: Maybe (Qualified Ident)`.